sdk/include/opentelemetry/sdk/metrics/export/periodic_exporting_metric_reader.h

```diff
@@ -0,0 +1,73 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#ifndef ENABLE_METRICS_PREVIEW
+
+#  include "opentelemetry/sdk/metrics/metric_reader.h"
+#  include "opentelemetry/version.h"
+
+#  include <atomic>
+#  include <chrono>
+#  include <condition_variable>
+#  include <thread>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace metrics
+{
+
+class MetricExporter;
+/**
+ * Struct to hold PeriodicExortingMetricReader options.
+ */
+
+constexpr std::chrono::milliseconds kExportIntervalMillis = std::chrono::milliseconds(60000);
```

Comment:
Are the default constant coming from the spec?

Reply:
yes, it is coming from here: https://github.com/open-telemetry/opentelemetry-specification/blob/99d14047268ff670ea2f3cacf597fa258bc4c11c/specification/metrics/sdk.md#periodic-exporting-metricreader